src/Swagger/Serializer/DocumentationNormalizer.php

```diff
@@ -230,6 +238,56 @@ private function updateGetOperation(\ArrayObject $pathOperation, array $mimeType
     }
 
     /**
+     * @param string $resourceClass
+     * @param string $operationName
+     *
+     * @return array
+     */
+    protected function getPaginationParameters(string $resourceClass, string $operationName)
```

Comment:
Missing return type, and should be `private` (we use `private` by default to ease the maintenance).